comp/virtual-comp/announce.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Announce the $SRYYYY virtual competition
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+area-owner: teams
+
+description: >-
+  The teams will need to know about the virtual competition, so we need to tell
+  them about it.
+
+  Required actions:
+   - [ ] Publish the schedule on the website
+   - [ ] Email teams about the virtual competition well ahead of time
+   - [ ] Remind teams about the virtual competition
+
+dependencies:
+  - comp/virtual-comp/plan
+  - comp/virtual-comp/code-submitter
```

Comment:
I don't think this task is needed. This should be a tickbox (or two) in the plan.yaml

Reply:
Could you elaborate on why you feel setting up the code submitter doesn't need to be a separate task?

Reasons to keep it separate:
- it will likely be done by different people than those doing the announcing
- it may involve a request to the infrastructure team for example for the instantiation of new hosting (for the year)
- it can (and ideally should) be completed well ahead of the announcement -- for example it's needed for the friendlies if we run them